config/routes.rb

```diff
@@ -2349,6 +2355,10 @@
         detach
         evacuate
         evacuate_form_fields
+        associate_floating_ip
```

Comment:
I am not sure here. Is there a need for associating FloatingIP to both Vm and VmCloud?